src/_guides/libraries/library-tour.md

```diff
@@ -5,19 +5,105 @@ description: "Learn about the major features in Dart's libraries."
 short-title: Library Tour
 ---
 
-This page shows you how to use the major features in Dart’s
-libraries. It’s just an overview, and by no means comprehensive.
-Whenever you need more details about a class, consult the [Dart API
-reference.][Dart API]
+This tour shows how to use the major features of the following libraries,
+which are included in all Dart platforms:
+
+{% comment %}
+[CHECK: is "all" guaranteed to stay true?]
+{% endcomment %}
+
+[dart:core](#dartcore---numbers-collections-strings-and-more)
+: Built-in types, collections, and other core functionality.
+  This library is automatically imported into every Dart program.
+
+[dart:async](#dartasync---asynchronous-programming)
+: Support for asynchronous programming, with classes such as Future and Stream.
+
+{% comment %}
+update-for-dart-2
+Q: When will Future move to dart:core?
+{% endcomment %}
+
+[dart:math](#dartmath---math-and-random)
+: Mathematical constants and functions, plus a random number generator.
+
+[dart:convert](#dartconvert---decoding-and-encoding-json-utf-8-and-more)
+: Encoders and decoders for converting between different data representations, including JSON and UTF-8.
+
+This page is just an overview;
+it doesn't include many dart:* libraries
```

Comment:
"it doesn't include many ... or any ..." --> "it excludes many ... and all ... "